docker-compose.yml

```diff
@@ -195,7 +195,7 @@ services:
         volumes:
             - ./api.raml:/api.raml
             - ./api.html:/api.html
-        command: -i api.raml -o api.html
+        command: mattjtodd/raml2html -i api.raml -o api.html
```

Comment:
Did you encounter a problem with this line? It works both ways for me.

Reply:
I did this while I was troubleshooting.  I think it helps (for future developers) to make the command explicit.  It's strange to me that it works without it.
The docs say `command overrides the default command declared by the container image (i.e. by Dockerfile’s CMD).`